lib/rbs/errors.rb

```diff
@@ -350,4 +350,42 @@ def location
       original.location
     end
   end
+
+  class MixinClassError < DefinitionError
+    attr_reader :type_name
+    attr_reader :member
+
+    def initialize(member:)
+      @type_name = member.name
+      @member = member
+
+      super "Cannot #{mixin_name} a class #{member.name} in the definition of #{type_name}"
+    end
+
+    def location
+      member.location
+    end
+
+    def self.check!(env:, member:)
```

Comment:
Add `type_name:` keyword argument.

Reply:
now error is like this
```
:in `check!': test.rbs:6:2...6:13: Cannot include a class `::Foo` in the definition of `::Qux` (RBS::MixinClassError)
```
this is the sample file
```ruby
class Foo
  def foo: () -> untyped
end

class Qux
  include Foo
end
```